src/app/components/SearchResults/Filters.jsx

```diff
@@ -157,105 +178,192 @@ class Filters extends React.Component {
-                  inputClassName="tablet:grid-col padding-right-4"
-                  className="grid-row"
+                <DS.DateRangeForm
+                  formLabel={<>Publication Year</>}
+
+                  fromLabelOpts={{ labelContent: <>From</>, id: 'FromLabel' }}
+                  fromInputOpts={{
+                    inputId: 'fromInput', inputValue: start, onInputChange: event => this.onChangeYear(event, 'start'),
+                  }}
+                  fromHelper={{ content: <>EX. 1901</>, id: 'fromyearhelper', isError: false }}
+
+                  toLabelOpts={{ labelContent: <>To</>, id: 'ToLabel' }}
+                  toInputOpts={{ inputId: 'toInput', inputValue: end, onInputChange: event => this.onChangeYear(event, 'end') }}
+                  toHelper={{ content: <>EX. 2000</>, id: 'toYearHelper', isError: false }}
+
+                  showError={this.state.error}
+                  error={{ content: <div>{this.state.errorMsg}</div>, id: 'date-range-error', isError: true }}
+
+                  buttonOpts={!isMobile
+                    ? { id: 'submitButtonId', callback: event => this.onSubmit(event, toggleMenu), content: <>Apply</> }
+                    : null}
                 />
               )}
               {field === 'show_all' && (
-                <Checkbox
-                  className="usa-checkbox"
-                  labelClass="usa-checkbox__label"
-                  inputClass="usa-checkbox__input"
-                  id="show_all"
+                <DS.Checkbox
+                  checkboxId="show_all"
                   isSelected={!this.isFilterChecked(field, true)}
                   onChange={e => this.onChangeCheckbox(e, field, true, true)}
-                  label="Available Online"
+                  labelOptions={{ id: 'show_all_label', labelContent: <>Available Online</> }}
                   name="show_all"
                 />
               )}
               {field === 'language'
-                && this.prepareFilters(data.facets[field], field).map(facet => (
-                  <Checkbox
-                    className="usa-checkbox"
-                    labelClass="usa-checkbox__label"
-                    inputClass="usa-checkbox__input"
-                    id={`filters-${field}-${facet.value}`}
-                    isSelected={this.isFilterChecked(field, facet.value)}
-                    onChange={e => this.onChangeCheckbox(e, field, facet.value, false)}
-                    label={facet.count > 0 ? `${facet.value} (${facet.count.toLocaleString()})` : `${facet.value}`}
-                    name={`filters.${field}`}
-                    key={`filters-${field}-${facet.value}`}
-                  />
-                ))}
+                && (
+                  <>
+                    {isMobile
+                      && (
+                      <DS.Accordion
+                        buttonOptions={{ id: 'accordionBtn', content: <span>Click to expand</span> }}
+                      >
```

Comment:
I'm gonna assign the list to a variable, which cleans it up in the code but functionally the component is still mounted twice (because from the filters perspective, it doesn't see the list, it just sees the accordion)